src/app/query-profile/query-profile.component.html

```diff
@@ -1,10 +1,12 @@
-<div *ngIf="dbServer === null" class="alert alert-warning w-75 mx-auto text-center wrap" role="alert">
+<div *ngIf="(dbServer === null) && (!isAllSelected) && (!isNotExistSelected)" class="alert alert-warning w-75 mx-auto text-center wrap" role="alert">
```

Comment:
brackets are rudimental here.

Reply:
I think it is more readable and prevent errors with prioritizing of logical operations for multiple conditions. What do you think regarding this case ?